### PR TITLE
Fix #653: settlement P&L reaches the scorecard — settle() close trades, backfill, drift repricing

### LINKED ISSUE #653: [CRITICAL] Data Integrity: P&L scorecard excludes reconcile-settled positions — reports +$377.50 while account implies ≈ -$431 MTM
Pro deep-analysis finding (2026-07-03), highest severity of the audit: every downstream consumer (Pro tuning, operator decisions, sizing confidence) is anchored to a scorecard ~$1,163 too optimistic.

**Evidence:**
- `gimmes report`: 31 closed trades (15W/14L/2S), gross +$526.93, net +$377.50.
- Paper account: $10,000 start; cash $7,985.55 + open cost basis $1,228.49 = $9,214 at cost ($9,569 MTM incl. +$355 unrealized) → implied realized P&L ≈ **-$786 at cost / -$431 MTM**.
- Gap ≈ $1,163, consistent with ~11-12 April/May positions (~$3,730 combined cost basis: KXCPI-26APR-T0.5, KXCPICORE-26APR-T0.3, KXADP-26APR, KXU3-26APR, KXCPIYOY-26APR-T3.7, KXUE-CAN, KXUE-UK, KXCPI-26MAR, 2x KXJOBLESSCLAIMS, KXGDP-26APR-T3.0) that settled via reconcile and have **no close records and no W/L attribution**.
- Same-table inconsistency: Total Trades 46 but W+L+scratch=31 and open=3 → 12 unaccounted.
- Likely related: #622/#628 excluded reconcile closes from DAILY P&L (correct — broker drift isn't a daily realized loss) but the exclusion appears to also swallow LIFETIME settlement P&L in the scorecard, where those settlements are real outcomes.

**Fix direction:** reconcile-settled positions must appear in the lifetime scorecard as settled outcomes (W/L + P&L) while staying excluded from the daily-loss trigger. Verify Largest Loss display while in there (report shows -$178.81; KXCPIYOY-26JUN-T3.9 computes ~-$183 from displayed prices).

### PR DESCRIPTION
**Pull request by jonnymobs907**
## Summary

**The bug:** `gimmes report` said +$377.50 net; the paper account implied ≈ −$431 MTM. Two causes, both verified against the live DB:
1. Paper-broker `settle()` credited the balance and zeroed positions but **wrote no close trade** — ~9 ghost settlements (~$1,050 realized) invisible to the scorecard.
2. The 7 reconcile drift closes that did exist were priced at **stale marks** — one recorded a +$5 "win" at mark $0.705 on a market that settled at $0.00 (a ~$366 loss).

## The fix

- **Forward:** `settle()` writes an `agent='settlement'` close at true settlement value (fee-free automatically via the fee curve's price-range guard), fills `resolved_outcome` (settlement *is* the resolution event — Monitor's log-outcome was proven wrong at least once), writes a `Trigger: Settlement` note (#586 lockout-safe), and deletes the positions mirror row in the same transaction. A residual-count duplicate-guard in `_log_reconcile_closes` backs it up — quantity-based and side-filtered after review showed a timestamp guard suppresses residual closes after partials, with direct kill-coverage tests after mutation testing showed zero coverage on the first draft.
- **Read-time:** reconcile drift closes reprice at settlement value when the group's resolution is known — the forward path for championship mode, where settlements arrive as drift. Settlement closes count in daily P&L (real money lost that day); **#622's drift exclusion untouched and pinned**.
- **One-time:** hidden `gimmes backfill-settlements` (`--dry-run`) inserts the missing closes at historical timestamps (isolated from today's daily-loss trigger — pinned) and repairs the 7 drift rows; idempotent; resolved_outcome conflicts prefer paper truth loudly.
- **Consistency:** `PnLSummary.open_trades` + Closed/Open rows — total = closed + open.

## Behavior change to note
Paper settlement losses now hit that day's daily P&L and can contribute to the daily-loss trigger — intended (they're real), but new for the autonomous loop.

## Review pipeline
3 reviewers (transaction atomicity, guard edge cases, mutation testing) + simplifier; all findings ≥80 applied. Follow-ups: #663 (authoritative settlements-API source, manual-close repricing caveat, resting-sell double-count).

## Test plan
- 129 targeted tests across 8 files (settle close rows incl. partial-sell residual, guard kill-coverage incl. reopen/partial/no-history, repricing matrix, backfill: ghost/idempotent/dry-run/repair/conflict/isolation/table-guard, daily-pnl inclusion+exclusion, formatter rows).
- Full suite: **1624 passed**; ruff clean.
- Post-merge operational step: run `gimmes backfill-settlements --dry-run`, eyeball the 12 candidates, then apply.

Closes #653

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01XraN34AP4re87cAzt9LRKB

### CODE REVIEW

File: src/gimmes/store/queries.py [hunk 1]
```diff
@@ -329,6 +337,137 @@ async def _log_reconcile_closes(
...
+    """Fill `resolved_outcome` (where NULL) on a ticker's trade rows.
+
+    Commit-less variant of `update_trade_outcome` — the caller manages
+    the transaction (#653).
+    """
+    await db.conn.execute(
+        "UPDATE trades SET resolved_outcome = ?"
+        " WHERE ticker = ? AND resolved_outcome IS NULL",
+        (outcome, ticker),
+    )
```
[the remark this answers is not in the document]
Comment (jonnymobs907) [reply src/gimmes/store/queries.py:402]: Fixed in a3a1963. `fill_resolved_outcome` is now authoritative — fills NULLs AND corrects conflicting values (idempotent when correct). Both callers are settlement-derived (settle path + backfill Phase 2), so overwrite is right for each; Monitor's non-authoritative `update_trade_outcome` stays NULL-only and can no longer clobber settlement truth. Second-pass review caught the first regression test being vacuous — it now seeds a real trades row and is mutation-verified against the old semantics.
